astromenace.appdata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As I noted about OpenRA, it would be nice to have [OARS](https://odrs.gnome.org/oars) here as well. It's also useful to have more detailed description, some screenshots and release info.

Reply:
Yeah, this needs more comprehensive AppData, otherwise people won't know what to install or why they should.

Reply:
fixed